### PR TITLE
[New Rule] Keychain Password Retrieval via Commandline

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/810

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+updated_date = "2020/01/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may collect the keychain storage data from a system to acquire credentials. Keychains are the built-in way
+for macOS to keep track of users' passwords and credentials for many services and features such as WiFi passwords,
+websites, secure notes, certificates, and Kerberos.
+"""
+false_positives = ["Trusted parent process accessing their respective application passwords."]
```
Comment (bm11100) [inline rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:13]:
```suggestion
false_positives = ["Trusted parent processes accessing their respective application passwords."]
```

File: rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Keychain Password Retrieval via Commandline"
```
Comment (bm11100) [inline rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:18]:
```suggestion
name = "Keychain Password Retrieval via Command Line"
```

Comment (bm11100) [reply rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:18]: consistency with other rules

File: rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+creation_date = "2020/01/06"
+maturity = "production"
+updated_date = "2020/01/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may collect the keychain storage data from a system to acquire credentials. Keychains are the built-in way
+for macOS to keep track of users' passwords and credentials for many services and features such as WiFi passwords,
+websites, secure notes, certificates, and Kerberos.
```
Comment (bm11100) [inline rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:11]:
```suggestion
Adversaries may collect keychain storage data from a system to in order to acquire credentials. Keychains are the built-in way
for macOS to keep track of users' passwords and credentials for many services and features, including Wi-Fi and
website passwords, secure notes, certificates, and Kerberos.
```

Comment (bm11100) [reply rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:11]: re lint

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

**Pull request merged**